test/performance/odp_timer_perf.c

```diff
@@ -761,6 +759,29 @@ static int set_cancel_mode_worker(void *arg)
 					ret = -1;
 					break;
 				}
+			}
+		}
+		cancel_c2 = odp_cpu_cycles();
+		cancel_diff += odp_cpu_cycles_diff(cancel_c2, cancel_c1);
+
+		set_c1 = odp_cpu_cycles();
+		for (i = 0; i < num_tp; i++) {
+			tp = global->timer_pool[i].tp;
```

Comment:
Splitting timer cancel and start calls into separate loops changes application dynamics quite a lot. Currently, timer cancel and start calls are back-to-back so that there is very little chance that start call would fail (due to current time being too near).

Maybe this cancel-loop + start-loop test case could be a new mode (-m) in the test application.